README.rst

```diff
@@ -391,6 +391,40 @@ In case you just want to change the colors used in a chart here's how to do it:
         }
     }
 
+``OPENWISP_MONITORING_METRICS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    | ``dict``    |
++--------------+-------------+
+| **default**: | ``{}``      |
++--------------+-------------+
+
+This setting allows to define additional metric configuration or to override
+the default metric configuration defined in
+``openwisp_monitoring.monitoring.metrics.DEFAULT_METRICS``.
+
+For example, if you want to change the field_name of
+``clients`` metric to ``wifi_clients`` you can use:
+
+.. code-block:: python
+
+    from django.utils.translation import ugettext_lazy as _
+
+    OPENWISP_MONITORING_METRICS = {
+        'clients': {
+            'label': _('Clients'),
+            'key': '{key}',
+            'field_name': 'wifi_clients',
+        },
+    }
+
+**NOTE**: In order to make use of ``{key}`` and ``{field}`` in your custom metric configuration,
+you need to define the name in the following format ``<key> <field>``.
```

Comment:
I don't understand this sentence, can you reword it? Also, shouldn't it be `field_name` instead of `field`?